docs/make_index.py

```diff
@@ -25,22 +25,7 @@ def write_index_rst(readme_file=None, write_file=None):
         main_path = os.path.dirname(os.path.dirname(os.path.abspath(inspect.stack()[0][1])))
         readme_file = os.path.join(main_path, 'README.md')
 
-    readme_md = pypandoc.convert_file(readme_file, 'md')
-
-    travis_str = 'https://travis-ci.org/RadioAstronomySoftwareGroup/pyuvsim.svg'
-    regex_travis = re.compile(travis_str)
-    loc_travis_start = re.search(regex_travis, readme_md).start()
-    loc_travis_end = re.search(regex_travis, readme_md).end()
-    end_branch_str = r'\)\]'
-    regex_end = re.compile(end_branch_str)
-    loc_branch_end = re.search(regex_end, readme_md).start()
-    branch_str = readme_md[loc_travis_end:loc_branch_end]
-
-    cover_str = 'https://coveralls.io/repos/github/RadioAstronomySoftwareGroup/pyuvsim/badge.svg'
-    regex_cover = re.compile(cover_str)
-    loc_cover_start = re.search(regex_cover, readme_md).start()
-    loc_cover_end = re.search(regex_cover, readme_md).end()
-
+    pypandoc.convert_file(readme_file, 'md')
```

Comment:
In my experience this doesn't work. pypandoc doesn't properly handle the badges. You'll need to look at readthedocs to verify this works (you can also test it locally by building the sphinx docs).

I now do this a little differently, and I think more cleanly, in pyuvdata, but it still requires cutting out the badges and replacing them with the ones that work in rst.

Reply:
So I didn't add this in -- it was there already (just removed an unused variable). Willing to fix it in this PR though, if you like.

I'm actually not sure why we don't just write the README in rst -- github displays this perfectly well, and it means we don't have to do any conversion.

Reply:
I'm sorry, I put the comment on the wrong line. What I'm objecting to is removing all the previous lines. Or are you saying they weren't being used? If that's the case it's either a mistake and the badges aren't rendering properly on readthedocs or pypandoc is now handling the badges properly.

Reply:
Yeah they weren't being used (I was a bit confused as well). They are rendering on readthedocs (on master), so I assume pypandoc is handling them fine now?

Reply:
Interesting. Ok, as long as they're rending well I'm fine with taking them out.

Reply:
I have also confirmed by running sphinx locally, and the badges show fine.